src/h2_stream.erl

```diff
@@ -656,3 +680,8 @@ validate_pseudos(_, DoneWithPseudos, _Found) ->
       DoneWithPseudos)
         andalso
         no_upper_names(DoneWithPseudos).
+
+handle_data(server, CB, Bin, CallbackState) ->
+    CB:on_receive_request_data(Bin, CallbackState);
+handle_data(client, CB, Bin, CallbackState) ->
+    CB:on_receive_response_data(Bin, CallbackState).
```

Comment:
I found there's a bug here: `on_receive_response_data` is not defined as a callback function and not defined in `chatterbox_static_stream` module